src/app/metadata/features/dataset-details/dataset-details.component.html

```diff
@@ -73,14 +73,25 @@ <h3>{{ datasetDetails().title }}</h3>
               @if (study().ega_accession) {
                 <a
                   mat-stroked-button
-                  class="mb-4 w-64 sm:float-right sm:mb-0"
+                  class="mb-4 ml-2 w-64 sm:float-right sm:mb-0"
                   [href]="'https://ega-archive.org/studies/' + study().ega_accession"
                   target="_blank"
                   rel="noopener noreferrer"
                   data-umami-event="Dataset Details Visit EGA Study Page Clicked"
                   ><mat-icon>open_in_new</mat-icon>Visit EGA Study Page</a
                 >
               }
+              @if (study().accession) {
```

Comment:
Should we maybe make a 'download study metadata' button for the summary as well?